src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<blockquote>The issue here is that, in the default delivery of this tutorial and as mentionned before</blockquote><p>-&gt; "mentioned"</p><p> </p><blockquote>Luckily for us (or because the problem was well thought of),</blockquote><p>I don't think I would keep this parenthesis, but if you do, at least put a smiley 😊&nbsp;</p><p> </p><blockquote>the fuel is almost accessible as, even if it is not returned by the <code>mtow_loop</code>, it is an output of a model main function.</blockquote><p>-&gt; "the fuel is almost accessible as it is an output of a model main function, even if it is not returned by the <code>mtow_loop.</code>"</p><p>(more readable, IMHO)</p><p> </p><blockquote>, but this will not always be the case.</blockquote><p>-&gt; ", but this may not always be the case." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Changed in dfe233f7